src/test_l1_hdmi_cec_driver.c

```diff
@@ -79,11 +79,9 @@
 */
 #define DEFAULT_LOGICAL_ADDRESS_STB 3
 
-
 /// Set the CEC sink (Display device) logical address here
 #define DEFAULT_LOGICAL_ADDRESS_PANEL 0
 
-
 static int gTestGroup = 1;
 static int gTestID = 1;
```

Comment:
in test_hdmicec_hal_l1_getPhysicalAddress_positive() test case 
according the header file 
https://github.com/rdkcentral/rdk-halif-hdmi_cec/blob/main/include/hdmi_cec_driver.h#L264
if(physicalAddress >= 0xffff){
	    UT_FAIL("Invalid physicalAddress ");
    }
For sink device 0.0.0.0 as the Physical Address    ?

Reply:
Checking the physical address value according to profile value.

Reply:
logic will be 
if(((strncmp(typeString,"source",UT_KVP_MAX_ELEMENT_SIZE) == 0) && ((physicalAddress<0x0000)||(physicalAddress >= 0xffff))) || \
            ((strncmp(typeString,"sink",UT_KVP_MAX_ELEMENT_SIZE) == 0) && (physicalAddress != 0x0000))){
	    UT_FAIL("Invalid physicalAddress");
    }

Reply:
updated logic. Earlier logic used as the comparison is for hex value. Did some  research and found hex value will be translated as singed int until we append u to value.
eg : physicalAddress >= 0xffffu)
Currently moved to the logic suggested.